File: c++/menor-de-tres.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;
int main() {

    int a, b, c, menor;

    cout << "Primeiro valor: ";
    cin >> a;
    cout << "Segundo valor: ";
    cin >> b;
    cout << "Terceiro valor: ";
    cin >> c;

    menor = 0;

    if (a < b && a < c)
    {
        menor = a;
    }
    else if (b < a && b < c)
    {
        menor = b;
    }
    else
    {
        menor = c;
    }
    
    cout << "MENOR = " << menor << endl;

    return 0;
}

File: python/menor-de-tres.py
a: int
b: int
c: int
menor: int

menor = 0

a = int(input("Primeiro valor: "))
b = int(input("Segundo valor: "))
c = int(input("Terceiro valor: "))

if a < b and a < c:
    menor = a
elif b < a and b < c:
    menor = b
else:
    menor = c

print(f"MENOR = {menor}")